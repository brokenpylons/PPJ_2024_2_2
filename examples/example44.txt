(3 * (x^2 + 5) / (2 * y)) - 8 * z / (2.5^3)
