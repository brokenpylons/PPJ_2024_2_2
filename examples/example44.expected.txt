lparen("(") real("3") times("*") lparen("(") variable("x") pow("^") real("2") plus("+") real("5") rparen(")") divides("/") lparen("(") real("2") times("*") variable("y") rparen(")") rparen(")") minus("-") real("8") times("*") variable("z") divides("/") lparen("(") real("2.5") pow("^") real("3") rparen(")") 
